input/docs/fundamentals/asynchronous-tasks.md

```diff
@@ -0,0 +1,25 @@
+Order: 55
+---
+
+# Asynchronous Tasks
+
+Cake supports having asynchronous tasks since version 0.23.0, this will let you use the `async` and `await` C# features within a task.
+
+This is really simplifies asynchronous programming within your Cake scripts.
```

Comment:
I think this needs to change...

`This really simplifies...`

Reply:
to what? :)
Perhaps:
`This makes it easier to do asynchronous programming within your Cake scripts.`
?

Reply:
You currently have `This is really simplifies...` and I was suggesting that you change it to `This really simplifies...`  i.e. remove the `is` as that part doesn't really make sense.

Reply:
Ah sorry :) I'll fix.